research/optimal_BERT_surgeon_oBERT/recipes/3epochs_unstructured90_mlm.yaml

```diff
@@ -0,0 +1,42 @@
+modifiers:
+  - !EpochRangeModifier
+    start_epoch: 0
+    end_epoch: 3
+
+training_modifiers:
+  - !LearningRateFunctionModifier
+    start_epoch: 0.0
+    end_epoch: 3.0
+    lr_func: cyclic_linear
+    cycle_epochs: 0.25
+    init_lr: 5e-4
+    final_lr: 5e-6
+
+  - !OBSPruningModifier
+    params: [
+      "re:bert.encoder.layer.*.attention.self.query.weight",
+      "re:bert.encoder.layer.*.attention.self.key.weight",
+      "re:bert.encoder.layer.*.attention.self.value.weight",
+      "re:bert.encoder.layer.*.attention.output.dense.weight",
+      "re:bert.encoder.layer.*.intermediate.dense.weight",
+      "re:bert.encoder.layer.*.output.dense.weight",
+    ]
+    init_sparsity: 0.7
+    final_sparsity: 0.9
+    start_epoch: 0
```

Comment:
@eldarkurtic Is start_epoch here actually 0 (instead of 0.25 to match with how the LR works)? Or this finetuning stage before pruning not needed for oBERT?

Reply:
@eldarkurtic yes that's what I thought, just wanted to confirm. Thanks.